models/object_detection/model_templates/custom-object-detection/readme.md

```diff
@@ -140,3 +140,39 @@ python eval.py \
    --save-metrics-to ${WORK_DIR}/metrics.yaml \
    --classes ${CLASSES}
 ```
+
+### 11. Optimization
+
+The trained models can be optimized -- compressed by [NNCF](https://github.com/openvinotoolkit/nncf) framework.
+
+At the moment, the following compression methods are supported:
+* int8 quantization
+* sparsity
```

Comment:
Sparsity and filter pruning was not added. 
I think that will be better to add link to compression_config.json.

Something like
```
List of available compression methods you can find in compression_config.json.
```

Reply:
PS. sparsity config exists only for face-detection-0200

Reply:
I suggest removing this config since it is not validated and planned to be used.